modules/processing/static.py

```diff
@@ -345,9 +365,17 @@ def run(self):
         results["pe_timestamp"] = self._get_timestamp()
         results["pdb_path"] = self._get_pdb_path()
         results["signature"] = self._get_signature()
+
+        sigverification = self._get_signature_verification()
+        if sigverification is not None:
+            results["signature_verification_verified"] = sigverification['verified']
+            results["signature_verification_output"] = sigverification["output"].replace('\r','')
+
```

Comment:
Yes, that's how I had it the first time, but I wasn't sure how to fix that in the html template to check on the existence of a sub key.  Check out the html index file and you may see what I mean.